### PR TITLE
Fix Default Port Value

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Default port value is returning NaN.

### CODE REVIEW

**Review by tbiethman — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Review by bjankord — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -2,4 +2,4 @@
 FROM cerner/terra-node
 
 # run the server
-CMD ["npm", "run", "start"]
+CMD ["npm", "run", "start", "--", "--port", "8080"]
```
Comment (mjhenkes) [inline Dockerfile]: This is temporary until we get a clean release right?

Comment (emilyrohrbough) [reply Dockerfile]: Yes should be. @ryanthemanuel When do we push the 'latest' tagged docker image?

Comment (mjhenkes) [reply Dockerfile]: Travis ci does it. https://github.com/cerner/terra-toolkit/blob/master/.travis.yml#L58

Comment (mjhenkes) [reply Dockerfile]: Though, i'd be remiss if i didn't point out that if we were testing against this version and not the previously released version, this issue wouldn't have been released.

**Comment by tbiethman** [comment]
+1 for commit messages

**Comment by bjankord** [comment]
Can we test this branch against terra-core?

**Review by yuderekyu — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]
+1 final changes

File: Dockerfile (hunk 1 above)
Comment (mjhenkes) [reply Dockerfile]: Also i'm pretty sure my previous comment was wrong. Please disregard. :)

**Comment by emilyrohrbough** [comment]
After spending way too log going down a lovely rabbit hole with @mjhenkes , it was discovered `har-validator` project released v5.1.3 three days ago and must have [corrupted/removed the v5.1.2 version](https://github.com/ahmadnassri/node-har-validator/issues/113). This is why the builds were continuously failing on the install step.

screen shot 2018-11-13 at 10 01 53 am


Removing the package-lock did the trick to clean this up.

**Pull request merged**